dev-python/pbr/pbr-5.11.1.ebuild

```diff
@@ -34,7 +34,7 @@ BDEPEND="
 			>=dev-python/fixtures-3.0.0[${PYTHON_USEDEP}]
 			>=dev-python/mock-2.0.0[${PYTHON_USEDEP}]
 			>=dev-python/six-1.12.0[${PYTHON_USEDEP}]
-			dev-python/sphinx[${PYTHON_USEDEP}]
+			<dev-python/sphinx-7[${PYTHON_USEDEP}]
```

Comment:
Any upstream bug or similar for the Sphinx thing? If not, please file one. If there is, please add a comment linking to it.